src/babylon.scene.ts

```diff
@@ -3220,6 +3232,7 @@
          */
         public addMaterial(newMaterial: Material): void {
             this.materials.push(newMaterial);
+            this.onNewMaterialAddedObservable.notifyObservers(newMesh);

```

Comment:
by the way what is "newMesh" here?

Reply:
done : b264afcfacee97923918acd32f1da27fa35ae244